### ISSUE TITLE
About Loss  and Accuracy ?

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [jimeffry/face-anti-spoofing#7]
Hi I'm Thai Hoc

I have a question for training model (I try to ROSE dataset which has 7 classes. But I merge all these classes to 2 new class is 'REAL' and 'FAKE'). But accuracy is very low at 50%.
I still use all model of your project But It's impossible 

I need to your help to improve my accuracy ? 
Thanks you for reading my question 

Have a good day 
Thai Hoc

**Comment by jimeffry** [comment]
The model of tensorflow is hard to convergence,because no pretrained model. I  recommend  the mxnet  model to you. Please read the first line in ReadMe.
